src/facade/conn_context.h

```diff
@@ -37,7 +37,7 @@ class ConnectionContext {
   // connection state / properties.
   bool conn_closing : 1;
   bool req_auth : 1;
-  bool replica_conn : 1;
+  bool replica_conn : 1;  // whether it's a replica connection on the master side.
```

Comment:
I dont see where this is set

Reply:
it's set here: https://github.com/dragonflydb/dragonfly/blob/main/src/server/server_family.cc#L2952

(not part of this PR)